src/core/connection.h

```diff
@@ -1403,6 +1404,22 @@ QuicConnQueueUnreachable(
     _In_ const QUIC_ADDR* RemoteAddress
     );
 
+//
+// Queues a route completion event to a connection for processing.
+//
+_IRQL_requires_max_(DISPATCH_LEVEL)
+_Function_class_(CXPLAT_ROUTE_RESOLUTION_CALLBACK)
+void
+QuicConnQueueRouteCompletion(
+    _Inout_ QUIC_CONNECTION* Connection,
+    _When_(Succeeded == FALSE, _Reserved_)
+    _When_(Succeeded == TRUE, _In_)
+    _In_reads_bytes_(6)
```

Comment:
```suggestion
    _When_(Succeeded == TRUE, _In_reads_bytes_(6))
```
The SAL isn't quite right. You'll need to update this everywhere I think.